{"name": "异常用例: 姓名-姓名, 账号-账号, 手机号-, 邮箱-abc@1122", "status": "passed", "attachments": [{"name": "测试图片", "source": "80e9b6ce-5dc5-4514-b80e-5faada8ba3a1-attachment.png", "type": "image/png"}], "parameters": [{"name": "get_add_member_fail_datas", "value": "['姓名', '账号', '', 'abc@1122', 'selenium.common.exceptions.TimeoutException']"}], "start": 1624631904964, "stop": 1624631916245, "uuid": "34e821e1-b1dd-43ca-969f-97598850b00d", "historyId": "0590e3a2609c642e61e038491477317d", "testCaseId": "352f8543e2d167406e331a06738aacaa", "fullName": "tests.test_wework.TestWework#test_add_member_fail", "labels": [{"name": "story", "value": "添加成员失败"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_wework"}, {"name": "subSuite", "value": "TestWework"}, {"name": "host", "value": "ccyangdeMacBook-Pro.local"}, {"name": "thread", "value": "68010-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_wework"}]}